solutions/usaco-1059.mdx

```diff
@@ -8,6 +8,27 @@ author: Sathvik Chundru
 [Official Analysis](http://www.usaco.org/current/data/sol_prob1_bronze_dec20.html)
 
 <LanguageSection>
+	
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	int nums[7];
```

Comment:
change to vector

Reply:
why?

Reply:
harder to peek inside

Reply:
not sure what you mean by that, it's fine as is

Reply:
it's just harder to print out arrays